src/controllers/informationController.js

```diff
@@ -56,7 +66,9 @@ const informationController = function (Information) {
         }
         res.status(200).send(deletedInformation);
       })
-      .catch((error) => res.status(400).send(error));
+      .catch((error) =>
+        res.status(400).send(new Error('Error when finding and deleting information by Id')),
+      );
```

Comment:
This is fine for a response to the frontend, but if we want to log these errors in production, wouldn't it help to use a logger?